packages/react/package.json

```diff
@@ -12,6 +12,11 @@
     "./hooks": {
       "types": "./dist/hooks.d.ts",
       "import": "./dist/hooks.mjs"
+    },
+    "./tw": {
```

Comment:
Did we wanna move this export to the main package?

Reply:
Yes, I think we should. Since it is not react-specific.